assets/header.css

```diff
@@ -380,7 +401,19 @@
   .header__content-inner {
     grid-template-columns: max-content 1fr max-content;
     grid-template-areas: "column-1 column-2 column-3";
-    gap: 0px 33px;
+    gap: 0px;
+  }
+
+  .header__content-inner.header__menu-position--centered {
+    grid-template-columns: 160px 1fr 160px;
+  }
+
+  .header__logo-wrapper:not(:empty) {
+    margin: 0 33px 0 0;
+  }
+
+  .header__links:not(:empty) {
+    margin: 0 0 0 33px;
```

Comment:
Won't it be clearer to use `margin-{{ side }}` here? So, `margin-right` and `margin-left`